src/octopeer-github/content/Types/ElementID.ts

```diff
@@ -4,48 +4,57 @@
  */
 class ElementID {
 
+    /**
+     * The 'Getters in this list are not really getters,
+     * they are the closest thing TypeScript has to offer as
+     * public static final fields.
+     */
     /** Instantiated ElementID for the MergePullRequestButton. */
-    public static get MERGEPRREQUEST() { return new ElementID(101); }
+    public static get MERGE_PULLREQUEST() { return new ElementID(101); }
     /** Instantiated ElementID for the ClosePullRequestButton. */
-    public static get CLOSEPRREQUEST() { return new ElementID(102); }
+    public static get CLOSE_PULLREQUEST() { return new ElementID(102); }
     /** Instantiated ElementID for the CancelInlineCommentButton. */
-    public static get CANCELINLINECOMMENT() { return new ElementID(103); }
+    public static get CANCEL_INLINE_COMMENT() { return new ElementID(103); }
     /** Instantiated ElementID for the ConfirmInlineCommentButton. */
-    public static get CONFIRMINLINECOMMENT() { return new ElementID(104); }
+    public static get CONFIRM_INLINE_COMMENT() { return new ElementID(104); }
     /** Instantiated ElementID for the CreateInlineCommentButton. */
-    public static get CREATEINLINECOMMENT() { return new ElementID(105); }
+    public static get CREATE_INLINE_COMMENT() { return new ElementID(105); }
     /** Instantiated ElementID for the EditPullRequestNameButton. */
-    public static get EDITPRNAME() { return new ElementID(106); }
+    public static get EDIT_PRNAME() { return new ElementID(106); }
```

Comment:
Why is there a difference between `EDIT_PRNAME` and `CREATE_PR_COMMENT` in the underscore after `PR`?